src/main/java/com/mindee/geometry/Polygon.java

```diff
@@ -0,0 +1,48 @@
+package com.mindee.geometry;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Polygon {
+  private List<Point> coordinates = new ArrayList<>();
+  private BoundingBox bbox;
```

Comment:
The boundingbox does not need to be a property of a polygon, it can exist without a polygon too.

Reply:
Yes of course.
But why this fact will prevent to use it as a property in a polygon ?

Reply:
Could just have a method that calculates it, either in helpers `utils.getBbox(polygon Polygon)` or directly in the class `polygon.getBbox()`.

Don't think we need to calculate it and set in RAM systematically.

Reply:
As discussed, remove this property, use utils to calculate the bbox.